### PR TITLE
Add date to relative time

### PR DESCRIPTION
**Pull request by kennyt**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @philipnrmn

File: src/js/utils/DateUtil.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ const DateUtil = {
     return moment.unix(ms).fromNow();
   },
 
+  dateToRelativeTime: function (date) {
```
Comment (rcorral) [inline src/js/utils/DateUtil.js]: Would be nice to have a test and to also have some JDoc saying that the `date` param needs to be an instance of `Date`

Comment (kennyt) [inline src/js/utils/DateUtil.js]: alright done

**Comment by philipnrmn** [comment]
I'm OK with the implementation, but what's the intention of this PR? Why not just call the `moment` method inline?

**Comment by philipnrmn** [comment]
I'm merging this, but I'm still curious about what it's for.

**Comment by kennyt** [comment]
@philipnrmn to test it and also if we ever wanted to stop using moment and use something else, we'd have to change it in one place

**Pull request merged**